registry/pkg-metadata-filter/src/index.ts

```diff
@@ -48,7 +48,6 @@ export function filterPkgMetadataByPublishDate (
       const candidateParsed = tryParseSemver(candidate)
       if (
         !candidateParsed ||
-        candidateParsed.major !== originalSemVer.major ||
```

Comment:
as I mentioned in the issue, I think we can do this only for the "latest" dist-tag. Not all.